pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+
+# mysqld options excluded from list
+# RV/HM 18.07.2017 Temporarily added to EXCLUDED_LIST: --binlog-group-commit-sync-delay due to hang issues seen in 5.7 with startup like --no-defaults --plugin-load=tokudb=ha_tokudb.so --tokudb-check-jemalloc=0 --init-file=/home/hrvoje/percona-qa/plugins_57.sql --binlog-group-commit-sync-delay=2047
+EXCLUDED_LIST=( --admin-address --admin-port --admin-ssl-ca --admin-ssl-capath --admin-ssl-cert --admin-ssl-cipher --admin-ssl-crl --admin-ssl-crlpath --admin-ssl-key --admin-tls-ciphersuites --admin-tls-version --basedir --bind-address --binlog-checksum --binlog-group-commit-sync-delay --caching-sha2-password-private-key-path --character-sets-dir --chroot --coredumper --datadir --debug --default-time-zone --disabled-storage-engines --event-scheduler --ft-stopword-file --general-log-file --init-connect --init-file --initialize --initialize-insecure --init-replica --innodb-data-file-path --innodb-data-home-dir --innodb-directories --innodb-doublewrite-dir --innodb-ft-aux-table --innodb-ft-server-stopword-table --innodb-ft-user-stopword-table --innodb-interpreter --innodb-interpreter-output --innodb-log-group-home-dir --innodb-page-size --innodb-parallel-doublewrite-path --innodb-tmpdir --innodb-redo-log-archive-dirs --innodb-temp-data-file-path --innodb-temp-tablespaces-dir --innodb-undo-directory --innodb-undo-tablespaces --internal-tmp-mem-storage-engine --keyring-migration-destination --keyring-migration-host --keyring-migration-password --keyring-migration-port --keyring-migration-socket --keyring-migration-source --keyring-migration-user --language --lc-messages-dir --log-bin-index --log-error --log-error-services --log-error-suppression-list --log-error-verbosity --log-isam --log-slow-filter  --log-slow-verbosity --log-tc --log-timestamps --myisam-block-size --myisam-data-pointer-size --myisam-max-sort-file-size --myisam-mmap-size --myisam-recover-options --myisam-repair-threads --myisam-sort-buffer-size --myisam-stats-method --myisam-use-mmap --mysqlx-bind-address --mysqlx-socket --mysqlx-ssl-ca --mysqlx-ssl-capath --mysqlx-ssl-cert --mysqlx-ssl-cipher --mysqlx-ssl-crl --mysqlx-ssl-crlpath --mysqlx-ssl-key --persist-only-admin-x509-subject --pid-file --plugin-dir --plugin-load --port --proxy-protocol-networks --relay-log-index --replica-skip-errors --replica-type-conversions --report-host --report-password --report-port --report-user --secure-file-priv --slow-query-log-always-write-time --slow-query-log-file --slow-query-log-use-global-control  --socket --ssl-ca --ssl-capath --ssl-cert --ssl-cipher --ssl-crl --ssl-crlpath --ssl-key --terminology-use-previous --tls-ciphersuites --tmpdir --transaction-write-set-extraction --utility-user --utility-user-dynamic-privileges --utility-user-password --utility-user-privileges --utility-user-schema-access --version-suffix)
```

Comment:
What is the reason for excluding event-scheduler ?

Reply:
Configured option event-scheduler

Reply:
FYI, most excluded options all had a reason behind them. Some had big reasons, other minors. It would require testing per option of interest to see it can be re-included in some way/fashion.